fighthealthinsurance/extralink_fetcher.py

```
@@ -472,6 +473,100 @@ async def _extract_html_text(self, content: bytes) -> str:
+        Returns:
+            Tuple of (extracted_text, doc_type)
+
+        Raises:
+            Exception: If fetch or extraction fails
+        """
+        if url_validator:
+            content, content_type, file_size = await self._fetch_url_safe(
+                url, url_validator
+            )
+        else:
+            content, content_type, file_size = await self._fetch_url(url)
```

Ruff (0.15.9)

[warning] Unpacked variable `file_size` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)